xml/http_servers.xml

```diff
@@ -4246,4 +4246,31 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:acme:mini_httpd:-"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Unspecified, UPnP/[\d\.]+, Unspecified$">
+    <description> UPNP server</description>
+    <example>Unspecified, UPnP/1.0, Unspecified</example>
+    <param pos="0" name="service.family" value="UPnP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^WNR2000v([0-9]) UPnP/[\d\.]+ miniupnpd/([\d\.]+)$">
+    <description>Netgear WNR2000v5 Router UPnP</description>
+    <example service.version="5" service.component.version="1.0">WNR2000v5 UPnP/1.0 miniupnpd/1.0</example>
+    <param pos="1" name="service.version"/>
```

Comment:
Is this a `service.version` or `hw.version`?

Reply:
switched to hw.version